src/examples.rs

```diff
@@ -208,20 +208,36 @@ pub fn tickets_sign_pod_builder(params: &Params) -> SignedPodBuilder {
     builder
 }
 
-pub fn tickets_pod_builder(params: &Params, signed_pod: &SignedPod, expected_event_id: i64, expect_consumed: bool, blacklisted_emails: &Value) -> MainPodBuilder {
+pub fn tickets_pod_builder(
```

Comment:
This is just a formatting change right?  Maybe we should include a rustfmt.toml in the repo to have a single formatting style. @arnaucube